sources/packages/backend/apps/api/src/route-controllers/student/student.controller.service.ts

```diff
@@ -113,14 +114,36 @@ export class StudentControllerService {
    * @param studentId student id to retrieve the data.
    * @returns student profile details.
    */
-  async getStudentProfile(studentId: number): Promise<StudentProfileAPIOutDTO> {
+  async getStudentProfile(studentId: number): Promise<StudentProfileAPIOutDTO>;
+  /**
+   * Get the student information that represents the profile.
+   * @param studentId student id to retrieve the data.
+   * @param options options:
+   * - `withSensitiveData` boolean option to return sensitive data such as SIN.
+   * @returns student profile details.
+   */
+  async getStudentProfile(
+    studentId: number,
+    options: { withSensitiveData: true },
```

Comment:
are we able to set default values inside options? I remember it was not working, and I had to do something like this 
![image](https://github.com/bcgov/SIMS/assets/77353155/e48d2fb7-b621-48e0-9240-4a7e39d26882)
correct me if Iam wrong